examples/box_game/box_game_spectator.rs

```diff
@@ -62,8 +62,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             2.0,
             TimerMode::Repeating,
         )))
-        .add_system(print_network_stats_system)
-        .add_system(print_events_system)
+        .add_systems(Startup, print_network_stats_system)
```

Comment:
Isn't this supposed to be Update, not Startup?

Reply:
Yes, you're right. Fixed.